third_party/CMakeLists.txt

```diff
@@ -101,6 +101,31 @@ target_include_directories(
 
 target_compile_options(lz4 PRIVATE -fPIC)
 
+include(ExternalProject)
+externalproject_add(
+        SIMDCompressionAndIntersection
+
+        SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/SIMDCompressionAndIntersection
+        BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/SIMDCompressionAndIntersection
+        CONFIGURE_COMMAND ""
+        BUILD_COMMAND bash -c "output=$(cd ${CMAKE_CURRENT_SOURCE_DIR}/SIMDCompressionAndIntersection && make 2>&1) || (printf \"$output\\n\" && false)"
+        INSTALL_COMMAND false  # install should never be called, this is a safe guard that fails if it is
+        STEP_TARGETS build
+)
+
+file(GLOB_RECURSE SIMDCAI_FILES ${CMAKE_CURRENT_SOURCE_DIR}/SIMDCompressionAndIntersection/*)
```

Comment:
Igittipfui ... I don't have any real experience with CMake and for now everything is fine. But is that "necessary" or just a hacked way to get it working quick?